### PR TITLE
feat(atarashi): Add Atarashi to FOSSology

### PR DESCRIPTION
**Pull request by hastagAB**
## Description
Integrating Atarashi with FOSSology. Currently, Atarashi works as a stand-alone scanner for various OSS licenses. It needs to be integrated with the main FOSSology Software to have better usage.

### Changes
Changed Ninka codebase according per the need to integrate Atarashi with FOSSology.

## How to test
Do a fresh installation of the FOSSology software from source.

### CODE REVIEW

**Comment by ag4ums** [comment]
@hastagAB can you please cherry-pick the commits from here
https://github.com/GMishx/fossology/commit/3b31c17996691c5b3beebe90506876efb5f361dc

**Comment by ag4ums** [comment]
@hastagAB please fix the failing test cases..

image

File: install/scripts/install-atarashi.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e 
+
+sudo apt-get install software-properties-common
+sudo add-apt-repository ppa:deadsnakes/ppa
+sudo apt-get update
+sudo apt-get install python>=3.6
+
```
Comment (ag4ums) [inline install/scripts/install-atarashi.sh]: this installation script can be move to agent mod_deps file(not in its current form), as different distro has different types of supported version. 
You can use this script as a ref https://github.com/fossology/fossology/blob/master/src/ojo/mod_deps

**Comment by GMishx** [comment]
Hello @hastagAB,
Did you get time to look into the the CI failures?

Do let us know if you need any kind of help.

**Comment by hastagAB** [comment]
The `make` command for agent and test was working fine. The CI shows error with the `atarashi` command.

**Pull request merged**